src/__stories__/button-story.tsx

```diff
@@ -42,10 +49,32 @@ type Args = {
     disabled: boolean;
     showSpinner: boolean;
     small: boolean;
+    action: string;
     newTab: boolean;
 };
 
-const href = 'https://example.com';
+const getButtonActionProps = (action: string, newTab: boolean) => {
+    return action === 'onPress'
+        ? {
+              onPress: () => {
+                  window.alert('pressed!');
+              },
+              href: undefined,
+              to: undefined,
```

Comment:
why undefined?, just don't add them to the object

Reply:
Old code that I forgot to remove. Thanks!